### ISSUE TITLE
Silverstripe 4

### ISSUE THREAD

**Issue by ggrune** [andrewhoule/silverstripe-photogallery#15]
The module is incompatible with Silverstripe version 4.0.0.

**Comment by andrewhoule** [comment]
Yeah, the current branch is only compatible with SS 3.*  I still have to create a new SS 4 branch.
